### ISSUE TITLE
Cannot find charm in charmhub via the search

### ISSUE THREAD

**Issue by aieri** [canonical/catalogue-k8s-operator#47]
The catalogue charm can only be accessed if you know the [correct URL](https://charmhub.io/catalogue-k8s). None of the following searches returned any result:
https://charmhub.io/?q=catalogue-k8s
https://charmhub.io/?q=catalogue
https://charmhub.io/?q=catalog

The last one is the one I had initially tried.
Ideally, all three keywords should lead to the charm page.

**Comment by sed-i** [comment]
@Lukewh, @0dysseas, seems like releasing a charm under stable does not got it listed.
What can we do about this?

**Comment by 0dysseas** [comment]
Has this charm gone through the standard process for listing ([example](https://discourse.charmhub.io/t/review-trino-k8s-charm-for-listing/12350))? I couldn't find any relevant thread in the charmhub forum.

**Comment by sed-i** [comment]
Thanks @0dysseas, I was not aware the discourse listing process is now official.
What's the starting point? Just copy-paste from the post you linked?

**Comment by sed-i** [comment]
@Lukewh I wonder if we could somehow increase the discoverability of the listing process from charmhub.io.
The `Publicise` tab on the `/listing` page for example?

**Comment by Lukewh** [comment]
> @Lukewh I wonder if we could somehow increase the discoverability of the listing process from charmhub.io. The `Publicise` tab on the `/listing` page for example?

Indeed, had the same thought @AnnSoMuller what do you think?

**Comment by 0dysseas** [comment]
@sed-i You just need to create a new forum post similar to the example I linked and someone from IS DevOps will see to it. Once the review process is over, you can ping me and I'll mark the charm as listed.

**Comment by AnnSoMuller** [comment]
yes, indeed It would improve the publisher experience and remove frustration to make the listing process discoverable.
@Lukewh let's discuss the best way to handle this.

**Comment by lucabello** [comment]
Request being followed by @simskij :

https://discourse.charmhub.io/t/review-catalogue-k8s-for-listing/17905/4
